### DISCUSSION TITLE
Mediana de sectores vacios

### DISCUSSION THREAD

**Discussion by GuilleStankeS** [IIC2133-PUC/2024-2#93] [Tarea 1] [answered]
Hola, si ocurre un evento MEDIANS, y hay sectores que no tienen pinguinos... que se hace en ese caso? no se cxonsideran?

Y cuanto ocurre FESTIVAL, y hay sectores que no tienen pinguinos, como se debe hacer ahi?

**Comment by Segismundios** [comment, marked as answer]
Holaa @GuilleStanke. Puedes asumir que en todo momento habrá al menos un pingüino en cada sector.
